ocm/__main__.py

```diff
@@ -234,6 +236,37 @@ def upload(parsed):
     )
 
 
+def add_cref(parsed):
+    raw = _parse_yaml_or_json(parsed.file)
+    component = raw['component']
+    component_references: list = component['componentReferences']
```

Comment:
you should not assume `componentReferences` already exists.

-> check if absent, and initialise as empty list